api/src/main/java/org/pmiops/workbench/reporting/insertion/ColumnValueExtractorUtils.java

```diff
@@ -1,18 +0,0 @@
-package org.pmiops.workbench.reporting.insertion;
-
-public class ColumnValueExtractorUtils {
-
-  /**
-   * Poor man's static enum method or extension method to allow grabbing a table property without
```

Comment:
wherever this was used, we could have simply referred to the name of the table.  so we do that now.